flank-scripts/src/main/kotlin/flank/scripts/utils/Serialization.kt

```diff
@@ -0,0 +1,12 @@
+package flank.scripts.utils
+
+import kotlinx.serialization.DeserializationStrategy
+import kotlinx.serialization.SerializationStrategy
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.JsonConfiguration
+
+private val json by lazy { Json(JsonConfiguration.Stable) }
+
+fun <T> T.toJson(serializationStrategy: SerializationStrategy<T>) = json.stringify(serializationStrategy, this)
```

Comment:
I tested it and it does not work properly for all cases. Also using it will force use to use only classes with `@Serializable` annotation.
The issue is that 
`Can't locate argument-less serializer for class Object. For generic classes, such as lists, please provide serializer explicitly.`
Also, I tested it with `Any` but with same effect ...